lib/iris/__init__.py

```diff
@@ -121,7 +121,7 @@ def callback(cube, field, filename):
 
 
 # Iris revision.
-__version__ = '2.1.0.dev0'
```

Comment:
This is a bit of an unusual step - it is because I'm moving master back to v2.0.x branch.